locale/en/docs/guides/debugging_getting_started.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Debugging - Getting Started
+layout: docs.hbs
+---
+
+# Debugging Guide
+
+The goal of this guide is to provide you with enough information to get started
+debugging your Node.js apps and scripts.
+
+## Enable Debugging
+
+### --debug
+
+When started with the **--debug** or **--debug-brk** switches, Node.js listens
+for debugging commands defined by the [V8 Debugging Protocol][] on a TCP port,
+by default `5858`. Any debugger client which speaks this protocol can connect
+to and debug the running process.
+
+A running Node.js process can also be instructed to start listening for
+debugging messages by signaling it with `SIGUSR1` (on Linux and OS X).
+
+### --inspect
+
+**NOTE**: The `--inspect` option and protocol are _experimental_ and may change.
+
+When started with the **--inspect** or **--inspect-brk** switches, Node.js
+listens for diagnostic commands defined by the [Inspector Protocol][]
+via websockets, by default at host and port 127.0.0.1:9229.
+
+Each Node process is marked with its own UUID similar in form to
+`0f2c936f-b1cd-4ac9-aab3-f63b0f33d55e`, and debugger clients must specify this
+ID to connect to the debugee. The full URL is
+`ws://127.0.0.1:9229/0f2c936f-b1cd-4ac9-aab3-f63b0f33d55e`, with the specific
+UUID changing for each target.
+
+Metadata about the target debuggee including its UUID can be retrieved via HTTP
+(not websockets) from `http://[host:port]/json/list` with host and port
+properly specified (default: 127.0.0.1:9229). This returns a JSON object like
+the following; use its `webSocketDebuggerUrl` as the connection string including
+UUID:
+
+```javascript
+{
+  "description": "node.js instance",
+  "devtoolsFrontendUrl": "chrome-devtools://devtools/bundled/inspector.html?experiments=true&v8only=true&ws=127.0.0.1:9229/0f2c936f-b1cd-4ac9-aab3-f63b0f33d55e",
+  "faviconUrl": "https://nodejs.org/static/favicon.ico",
+  "id": "0f2c936f-b1cd-4ac9-aab3-f63b0f33d55e",
+  "title": "node",
+  "type": "node",
+  "url": "file://",
+  "webSocketDebuggerUrl": "ws://127.0.0.1:9229/0f2c936f-b1cd-4ac9-aab3-f63b0f33d55e"
+}
+```
+
+<!-- refs -->
+
+[V8 Debugging Protocol]: https://github.com/v8/v8/wiki/Debugging-Protocol
+[Inspector Protocol]: https://chromedevtools.github.io/debugger-protocol-viewer/v8/
+
+---
+
+## Debugging Clients
+
+Several commercial and open source tools can connect to Node's
+debugger and/or inspector. Info on these follows.
+
+* [Built-in Debugger](https://github.com/nodejs/node/blob/master/lib/_debugger.js)
```

Comment:
I think another way to organize things is:

```
### Inspector Protocol

* node-inspect
* Chrome DevTools
* VSCode

### Debugger Protocol

* Built-in Debugger
* Chrome DevTools
* VSCode
```

So people can first look for a client that uses the protocol they prefer(hopefully the inspector protocol) and look for alternatives if their current tool doesn't support the new protocol, but this works fine too.